### PR TITLE
DOC-3105: Low-Cost Fixed Plan and DOC-3107: Add that 7.2 is available in us-east-1 to November changelog

### PR DESCRIPTION
**Pull request by cmilesb**
DOC-3105 DOC-3107

Staging links: 
https://docs.redis.com/staging/DOC-3105/rc/subscriptions/create-fixed-subscription/
https://docs.redis.com/staging/DOC-3105/rc/changelog/november-2023/

### CODE REVIEW

**Comment by cmilesb** [comment]
Approved by Adi Abiri Silberman via Slack.

File: content/rc/changelog/november-2023.md [hunk 1]
```diff
@@ -2,17 +2,32 @@
...
 aliases: []
 ---
 
+## New features
+
+### New Fixed plans
+
+Redis is updating our [fixed subscriptions]({{<relref "/rc/subscriptions/create-fixed-subscription">}}) on Redis Cloud. These plans offer increased capacity at lower prices compared to legacy plans.
+
+See [Fixed plan subscription tiers]({{<relref "/rc/subscriptions/create-fixed-subscription#fixed-plan-subscription-tiers">}}) to see an updated list of all Fixed plans.
```
Comment (rrelledge) [inline content/rc/changelog/november-2023.md:18]:
```suggestion
See [Fixed plan subscription tiers]({{<relref "/rc/subscriptions/create-fixed-subscription#fixed-plan-subscription-tiers">}}) for an updated list of all Fixed plans.
```

File: content/rc/changelog/november-2023.md [hunk 2]
```diff
@@ -2,17 +2,32 @@
...
+
+### Redis 7.2 Fixed region support
+
+You can now use Redis 7.2 on [fixed subscriptions]({{<relref "/rc/subscriptions/create-fixed-subscription">}}) in `us-east-1` on Amazon Web Services. New fixed subscriptions created in `us-east-1` will be created with Redis 7.2, and existing subscriptions will continue with Redis 6.2.
```
Comment (rrelledge) [inline content/rc/changelog/november-2023.md:22]: Maybe?
```suggestion
You can now use Redis 7.2 on [Fixed subscriptions]({{<relref "/rc/subscriptions/create-fixed-subscription">}}) in `us-east-1` on Amazon Web Services. New Fixed subscriptions created in `us-east-1` will use Redis 7.2, and existing subscriptions will continue to use Redis 6.2.
```

File: content/rc/changelog/november-2023.md [hunk 3]
```diff
@@ -2,17 +2,32 @@
...
 alwaysopen: false
 categories: ["RC"]
 aliases: []
 ---
 
+## New features
+
+### New Fixed plans
+
+Redis is updating our [fixed subscriptions]({{<relref "/rc/subscriptions/create-fixed-subscription">}}) on Redis Cloud. These plans offer increased capacity at lower prices compared to legacy plans.
```
Comment (rrelledge) [inline content/rc/changelog/november-2023.md:16]:
```suggestion
Redis is updating our [Fixed subscriptions]({{<relref "/rc/subscriptions/create-fixed-subscription">}}) on Redis Cloud. These plans offer increased capacity at lower prices compared to legacy plans.
```

File: content/rc/subscriptions/create-fixed-subscription.md [hunk 4]
```diff
@@ -14,27 +14,25 @@ Fixed Size subscription plans support low throughput workflows.  Several tiers a
...
-|--------------------------------------:|:---------:|:-----------:|:---------------:|:---------------------:|
-| (Free) 30 MB &nbsp;&nbsp;&nbsp;&nbsp; | 1 | 30 | N/A |          N/A          |
-|       100 MB &nbsp;&nbsp;&nbsp;&nbsp; | 4 | 256 | 1 |           4           |
-|       250 MB &nbsp;&nbsp;&nbsp;&nbsp; | 8 | 256 | 1 |           4           |
-|       500 MB &nbsp;&nbsp;&nbsp;&nbsp; | 12 | 512 | 1 |           4           |
-|         1 GB &nbsp;&nbsp;&nbsp;&nbsp; | 16 | 1024 | 2 |           8           |
-|      2.5 GB &nbsp;&nbsp;&nbsp;&nbsp; | 24 | 2500 | 2 |           8           |
-|         5 GB &nbsp;&nbsp;&nbsp;&nbsp; | 32 | 5000 | 4 |          16           |
-|        10 GB &nbsp;&nbsp;&nbsp;&nbsp; | 64 | 10000 | 4 |          32           |
+| **Max DB Size &nbsp;** | **Number of<br/>Databases** | **Concurrent<br/>Connections<br/>per Database** | **CIDR<br/> Allow rules** |
```
Comment (rrelledge) [inline content/rc/subscriptions/create-fixed-subscription.md:24]:
```suggestion
| **Max DB size &nbsp;** | **Number of<br/>databases** | **Concurrent<br/>connections<br/>per database** | **CIDR<br/> allow rules** |
```

File: content/rc/subscriptions/create-fixed-subscription.md [hunk 5]
```diff
@@ -14,27 +14,25 @@ Fixed Size subscription plans support low throughput workflows.  Several tiers a
...
+| 250 MB &nbsp;&nbsp;&nbsp;&nbsp; | 8 | 256 | 4-8 |
+| 1 GB &nbsp;&nbsp;&nbsp;&nbsp; | 16 | 1024 | 4-8 |
+| 2.5 GB &nbsp;&nbsp;&nbsp;&nbsp; | 24 | Unlimited | 4-8 |
+| 5 GB &nbsp;&nbsp;&nbsp;&nbsp; | 32 | Unlimited | 4-16 |
+| 12 GB &nbsp;&nbsp;&nbsp;&nbsp; | 64 | Unlimited | 4-32 |
 
 The 30 MB Fixed plan is free; it's designed for training and prototype purposes.
 
-All paid (100 MB and above) fixed plans support replication and backups (daily and instant).
+All paid (250 MB and above) fixed plans support replication and backups (daily and instant).
```
Comment (rrelledge) [inline content/rc/subscriptions/create-fixed-subscription.md:35]:
```suggestion
All paid (250 MB and above) Fixed plans support replication and backups (daily and instant).
```

**Review by rrelledge — APPROVED** [review]
Looks good to me. I just left a few small suggestions.

**Pull request merged**